.codexplus/hooks/add_context.py

```
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Ruff (0.13.1)

Shebang is present but file is not executable

(EXE001)